setEnvVars.js

```diff
@@ -1,4 +1,4 @@
-process.env.AWS_STACK_NAME = "di-ipv-cri-toy-api";
+process.env.AWS_STACK_NAME = "ipv-cri-toy-api";
```

Comment:
Something to pull out for another PR - but do we even need this file, or can we use [dotenv](https://www.npmjs.com/package/dotenv).

If we do need this file, let's add a comment at the top to stop me asking this question again in 3 months time.